data.sql

```diff
@@ -1,5 +1,15 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_g) VALUES ('Agumon','2020-02-03',0,true,10.23);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_g) VALUES ('Gabumon','2018-11-15',2,true,8);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_g) VALUES ('Pikachu','2021-01-07',1,false,15.04);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_g) VALUES ('Devimon','2017-05-12',5,true,11);
+
+-- updated data
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Charmander','2020-02-08', 0, FALSE, -11 );
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Plantmon','2021-11-15', 2, TRUE, -5.7 );
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Squirtle','1994-04-02', 3, FALSE, -12.13 );
```

Comment:
- [ ] The year of birth is for `Squirtle` is `1993` and not `1994` kindly fix this.